### PR TITLE
Fix cucumber properties

### PR DESCRIPTION
**Pull request by carlogeenen**
Changed the filepath to the correct .properties files and SoapUI .xml project file to fix Cucumber testing.

### CODE REVIEW

**Comment by CLAAdmin** [comment]
[CLA assistant check](http://52.16.237.165:80/OSGP/Integration-Tests?pullRequest=81) <br/>All committers of the pull request should sign our Contributor License Agreement in order to get your pull request merged.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: carlogeenen<br/>:x: web-flow

File: integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/dbsupport/AbstractPersistenceConfig.java [hunk 1]
```diff
@@ -23,14 +23,14 @@
...
- * configuration requires Spring Framework 3.0
+ * Base class for PersistenceConfig classes. An application context Java JPA
+ * configuration class. The usage of Java configuration requires Spring
+ * Framework 3.0
  */
 @Configuration
 @EnableTransactionManagement()
 @Primary
-@PropertySource("file:/etc/osp/osgp-cucumber-response-data-smart-metering.properties")
+@PropertySource("file:/etc/osp/cucumber-platform-dlms.properties")
```
Comment (robindenadel) [inline integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/dbsupport/AbstractPersistenceConfig.java]: handle the property source the same way as in SLIM-686

File: integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/dbsupport/ApplicationContext.java [hunk 2]
```diff
@@ -20,10 +20,10 @@
...
 @ComponentScan(basePackages = { "com.alliander.osgp.adapter.ws.smartmetering.domain.entities",
         "com.alliander.osgp.domain.core.repositories", "com.alliander.osgp.domain.core.entities",
-"com.alliander.osgp.logging.domain.repositories" })
+        "com.alliander.osgp.logging.domain.repositories" })
 @EnableTransactionManagement()
 @Import({ PersistenceConfigCore.class, PersistenceConfigResponseData.class, PersistenceConfigResponseDlms.class,
-    PersistenceConfigLogging.class })
-@PropertySource("file:/etc/osp/osgp-cucumber-response-data-smart-metering.properties")
+        PersistenceConfigLogging.class })
+@PropertySource("file:/etc/osp/cucumber-platform-dlms.properties")
```
Comment (robindenadel) [inline integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/dbsupport/ApplicationContext.java]: handle the property source the same way as in SLIM-686

File: integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/dbsupport/PersistenceConfigResponseDlms.java [hunk 3]
```diff
@@ -22,13 +22,11 @@
...
 
-@EnableJpaRepositories(entityManagerFactoryRef = "entityMgrFactDlms", 
-    transactionManagerRef = "txMgrDlms",
-    basePackageClasses = { DlmsDeviceRepository.class })
+@EnableJpaRepositories(entityManagerFactoryRef = "entityMgrFactDlms", transactionManagerRef = "txMgrDlms", basePackageClasses = { DlmsDeviceRepository.class })
 @Configuration
 @EnableTransactionManagement()
 @Primary
-@PropertySource("file:/etc/osp/osgp-cucumber-response-data-smart-metering.properties")
+@PropertySource("file:/etc/osp/cucumber-platform-dlms.properties")
```
Comment (robindenadel) [inline integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/dbsupport/PersistenceConfigResponseDlms.java]: handle the property source the same way as in SLIM-686

File: integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/hooks/AddDeviceHooks.java [hunk 4]
```diff
@@ -29,7 +29,7 @@
  */
 @Component
 @Configuration
-@PropertySource("file:/etc/osp/osgp-cucumber-response-data-smart-metering.properties")
+@PropertySource("file:/etc/osp/cucumber-platform-dlms.properties")
```
Comment (robindenadel) [inline integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/hooks/AddDeviceHooks.java]: handle the property source the same way as in SLIM-686

File: integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/hooks/DeviceHooks.java [hunk 5]
```diff
@@ -23,19 +23,19 @@
  */
 @Component
 @Configuration
-@PropertySource("file:/etc/osp/osgp-cucumber-response-data-smart-metering.properties")
+@PropertySource("file:/etc/osp/cucumber-platform-dlms.properties")
```
Comment (robindenadel) [inline integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/hooks/DeviceHooks.java]: handle the property source the same way as in SLIM-686

File: integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/hooks/ScenarioHooks.java [hunk 6]
```diff
@@ -34,7 +34,7 @@
  * class for preparing specific scenarios
  *
  */
-@PropertySource("file:/etc/osp/osgp-cucumber-response-data-smart-metering.properties")
+@PropertySource("file:/etc/osp/cucumber-platform-dlms.properties")
```
Comment (robindenadel) [inline integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/hooks/ScenarioHooks.java]: handle the property source the same way as in SLIM-686

File: integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/support/ResponseNotifierImpl.java [hunk 7]
```diff
@@ -24,13 +24,13 @@
 
 @Component
 @Configuration
-@PropertySource("file:/etc/osp/osgp-cucumber-response-data-smart-metering.properties")
+@PropertySource("file:/etc/osp/cucumber-platform-dlms.properties")
```
Comment (robindenadel) [inline integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/support/ResponseNotifierImpl.java]: handle the property source the same way as in SLIM-686

File: integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/dbsupport/AbstractPersistenceConfig.java (hunk 1 above)
Comment (carlogeenen) [inline integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/dbsupport/AbstractPersistenceConfig.java]: ok

File: integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/dbsupport/ApplicationContext.java (hunk 2 above)
Comment (carlogeenen) [inline integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/dbsupport/ApplicationContext.java]: ok

File: integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/dbsupport/PersistenceConfigResponseDlms.java (hunk 3 above)
Comment (carlogeenen) [inline integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/dbsupport/PersistenceConfigResponseDlms.java]: ok

File: integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/hooks/AddDeviceHooks.java (hunk 4 above)
Comment (carlogeenen) [inline integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/hooks/AddDeviceHooks.java]: ok

File: integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/hooks/DeviceHooks.java (hunk 5 above)
Comment (carlogeenen) [inline integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/hooks/DeviceHooks.java]: ok

File: integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/hooks/ScenarioHooks.java (hunk 6 above)
Comment (carlogeenen) [inline integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/hooks/ScenarioHooks.java]: ok

File: integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/support/ResponseNotifierImpl.java (hunk 7 above)
Comment (carlogeenen) [inline integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/support/ResponseNotifierImpl.java]: ok

File: integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/hooks/ScenarioHooks.java [hunk 8]
```diff
@@ -34,7 +34,12 @@
  * class for preparing specific scenarios
  *
  */
-@PropertySource("file:/etc/osp/cucumber-platform-dlms.properties")
+// @PropertySources({
+// @PropertySource("classpath:osgp-cucumber-response-data-smart-metering.properties"),
+// @PropertySource(value =
+// "classpath:osgp-cucumber-response-data-smart-metering-${env}.properties",
+// ignoreResourceNotFound = true)}
+// )
```
Comment (robindenadel) [inline integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/hooks/ScenarioHooks.java]: Code block should not be commented out. If it is not needed, it should be removed

**Pull request merged**